diff --git a/loader/dist/vanilla-extract-webpack-plugin-loader.cjs.dev.js b/loader/dist/vanilla-extract-webpack-plugin-loader.cjs.dev.js
index 6ba499ea04282c29a05a5ae7616bc214371c52eb..4219ee66afb938701c52115d390a3db96c225134 100644
--- a/loader/dist/vanilla-extract-webpack-plugin-loader.cjs.dev.js
+++ b/loader/dist/vanilla-extract-webpack-plugin-loader.cjs.dev.js
@@ -33,14 +33,12 @@ function loader (source) {
     name
   } = integration.getPackageInfo(this.rootContext);
   const callback = this.async();
-  integration.transform({
-    source,
+  integration.compile({
     filePath: this.resourcePath,
-    rootPath: this.rootContext,
-    packageName: name,
+    cwd: this.rootContext,
     identOption: defaultIdentifierOption(this.mode, identifiers)
-  }).then(code => {
-    callback(null, code);
+  }).then(result => {
+    callback(null, result.source);
   }).catch(e => {
     callback(e);
   });
@@ -65,7 +63,6 @@ function pitch() {
     source
   }) => {
     const result = await integration.processVanillaFile({
-      source,
       outputCss,
       filePath: this.resourcePath,
       identOption: defaultIdentifierOption(this.mode, identifiers),
diff --git a/loader/dist/vanilla-extract-webpack-plugin-loader.cjs.prod.js b/loader/dist/vanilla-extract-webpack-plugin-loader.cjs.prod.js
index 8d9e8d2665f32adc85646c766d876543d26b942f..eaaa6ef621b44fea0a7a17709f522f4514be4b28 100644
--- a/loader/dist/vanilla-extract-webpack-plugin-loader.cjs.prod.js
+++ b/loader/dist/vanilla-extract-webpack-plugin-loader.cjs.prod.js
@@ -33,14 +33,12 @@ function loader (source) {
     name
   } = integration.getPackageInfo(this.rootContext);
   const callback = this.async();
-  integration.transform({
-    source,
+  integration.compile({
     filePath: this.resourcePath,
-    rootPath: this.rootContext,
-    packageName: name,
+    cwd: this.rootContext,
     identOption: defaultIdentifierOption(this.mode, identifiers)
-  }).then(code => {
-    callback(null, code);
+  }).then(result => {
+    callback(null, result.source);
   }).catch(e => {
     callback(e);
   });
@@ -65,7 +63,6 @@ function pitch() {
     source
   }) => {
     const result = await integration.processVanillaFile({
-      source,
       outputCss,
       filePath: this.resourcePath,
       identOption: defaultIdentifierOption(this.mode, identifiers),
